tests/e2e/cluster_test.go

```diff
@@ -290,6 +290,7 @@ func (cfg *etcdProcessClusterConfig) etcdServerProcessConfigs() []*etcdServerPro
 	for i := range etcdCfgs {
 		etcdCfgs[i].initialCluster = strings.Join(initialCluster, ",")
 		etcdCfgs[i].args = append(etcdCfgs[i].args, initialClusterArgs...)
+		etcdCfgs[i].args = append(etcdCfgs[i].args, "--logger=zap")
```

Comment:
@jingyih Should we make `zap` default in a separate PR, so we don't make this unrelated change?